packages/dds/tree/src/feature-libraries/modular-schema/genericFieldKind.ts

```diff
@@ -78,14 +82,20 @@ export const genericChangeHandler: FieldChangeHandler<GenericChangeset> = {
                     } else {
```

Comment:
Don't we need to call `composeChildren` on `nodeChange` for the above two branches as well? (I can't leave a comment on them, this is the closes I can get)